mockwebserver/src/main/kotlin/okhttp3/mockwebserver/internal/duplex/MockWebServerAsserts.kt

```diff
@@ -0,0 +1,31 @@
+package okhttp3.mockwebserver.internal.duplex
+
+open class MockWebServerAsserts {
```

Comment:
open?!

Reply:
Totally not understanding why this is a class and not an object

Reply:
It's an override mechanism.  The other implementation delegates to junit

Reply:
From the original comment, happy to find alternate design that by default works sensibly, but allows reverting back to using the junit API methods by override.